site/public/apps/rheum-derm-immune-atlas/index.html

```
@@ -0,0 +1,646 @@
+  <section id="sources" class="view">
+    <div class="grid two">
+      <section class="panel"><div class="panel-head"><div><h3>Evidence grading framework</h3><div class="panel-sub">A pragmatic hybrid of intervention evidence and mechanistic human validation.</div></div></div><div id="methodGrades" class="method-grid"></div></section>
+      <section class="panel"><div class="panel-head"><div><h3>Scope and limitations</h3><div class="panel-sub">Important boundaries for educational use.</div></div></div><div class="disclaimer"><b>Not medical advice.</b> This atlas omits dosing algorithms, contraindication detail, pregnancy management, vaccination schedules and many rare therapies. “Pathway importance” is a structured synthesis, not a validated biomarker score. “Benefit” is ordinal and should not be used for indirect treatment ranking across trials.</div><ul class="muted"><li>Rare-disease rows necessarily rely more on cohorts and case series.</li><li>Off-label and investigational use is labeled in mechanism/caveat text.</li><li>Regulatory status varies by country and can change after the evidence cutoff.</li><li>Organ-threatening disease requires disease-specific guideline and specialist review.</li></ul></section>
+    </div>
+    <section class="panel" style="margin-top:16px"><div class="panel-head"><div><h3>Source library</h3><div class="panel-sub">Guidelines, pivotal trials, regulatory sources and mechanistic reviews used for the synthesis.</div></div><div class="controls"><input id="refSearch" class="control" type="search" placeholder="Search references"><select id="refKind" class="control"><option value="">All source types</option></select></div></div><div id="referenceList" class="reference-list"></div></section>
+  </section>
+
+  <footer class="footer"><div>Rheum–Derm Immune Pathways & Therapeutics Atlas · Version <span id="version"></span> · Evidence cutoff <span id="footerCutoff"></span></div><div>Single-file offline application. External links open source searches; speech synthesis depends on browser support.</div></footer>
+</main>
+<div id="tooltip" class="tooltip"></div>
+<div id="modal" class="modal" role="dialog" aria-modal="true" aria-labelledby="modalTitle"><div class="modal-card"><div class="modal-head"><h3 id="modalTitle"></h3><button class="modal-close" aria-label="Close">×</button></div><div id="modalBody"></div></div></div>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)